src/assets/src/components/pageTitle.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Helmet } from "react-helmet-async";
+
+const BaseTitle = "Remote Office Hours Queue";
+
+export const createTitle = (title: string) => {
+    return `${title} - ${BaseTitle}`;
```

Comment:
Thanks this is good. I'd commented on a call that I watched a a11y training on[ Importance of Page Titles ](https://www.linkedin.com/learning-login/share?account=42573940&forceAccount=false&redirect=https%3A%2F%2Fwww.linkedin.com%2Flearning%2Faccessibility-for-web-design%2Fthe-importance-of-page-titles-14871242%3Ftrk%3Dshare_video_url%26shareId%3DHyhZR05JSUiakrUfesIYVA%253D%253D)

And the takeaway was

**Specific to General:** Titles should be structured from specific to general (e.g., "Billing information, checkout, Merch Warehouse") to help users quickly confirm they are on the correct page.

So this was reformatted for that.